app/controllers/verify/confirmations_controller.rb

```diff
@@ -13,6 +14,10 @@ def index
 
     private
 
+    def confirm_profile_present
```

Comment:
what does it mean to have a profile present? I can see that this looks at the `idv_session.profile` but perhaps a different name would clarify intent of this `before_action`?